### ISSUE TITLE
Didn't reproduce the result

### ISSUE THREAD

**Issue by iamGavinZhou** [Jongchan/tensorflow-vdsr#5]
I run you code, but i didn't reproduce the result. I use Tensorflow 0.12 with CUDA8, but there something run in your code, i just do a simple modification to  make it run.
I finally got the PSNR value on Set5 and Set14, far below the value of your paper.
On Set5:
```
x2 6 33.6565127209
x3 10 30.380419877
x4 10 28.3931595153
```
On Set14:
```
x2 6 30.2294985089
x3 102 27.5454416367
x4 6 26.0019467506
```
It's there anything wrong? Thanks

**Comment by Jongchan** [comment]
- I am not the author of the paper.. this is just a simple implementation of the paper.
- I am using the same framework as you, since I am using the latest docker image from Tensorflow. So it should not be a problem with the environment..

- It looks like the values are very close to the bicubic result.
- Have you uses the checkpoint file provided in the README file? How is the result?

**Comment by iamGavinZhou** [comment]
No, i didn't use the checkpoint file provided in the README file. I just train the model and do evaluate.

**Comment by Jongchan** [comment]
@GavinZhou2014 It seems the network is not trained at all. I need more information to figure this out. Does the loss go down? What modification did you make? How many epochs did you train?

I am kinda busy for a close deadline, so I will get back to you next week.

**Comment by harungunaydin** [comment]
Using AdamOptimizer with 1e-4 learning rate improves the results drastically. And it does not require gradient clipping either. I know it's not what they did in the paper but I'm just saying.

**Comment by Jongchan** [comment]
@harungunaydin The author stated they used 0.9 momentum, so I thought they used MomentumOptimizer. Thank you for pointing out new optimizer. I will certainly try that out and add it into the repository.
